arangosh/Import/ImportHelper.h

```diff
@@ -348,6 +360,9 @@ class ImportHelper {
 
   std::unordered_map<std::string, std::string> _translations;
   std::unordered_map<std::string, std::string> _datatypes;
+
+  std::vector<std::pair<std::string, std::vector<AttrProperties>>> _attrsToValues;
```

Comment:
I think this instance variable should have a more descriptive name. From the variable name alone, it is unclear that there is a connection to the `--merge-attributes` functionality. can we name it somehow that the connection is clear?